pkg-tsconfig/tsconfig.base.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should this extend from `./tsconfig.json`?

Reply:
> Should this extend from `./tsconfig.json`?

I left `tsconfig.json` alone because other config files were for building commonJS/ES modules.
`tsconfig.json` can be used for assert testing and jest